helpers/deploymentConfig.ts

```diff
@@ -12,14 +11,22 @@ export enum InterestRateModels {
   JumpRate,
 }
 
-export type TokenConfig = {
-  isMock: boolean;
-  name?: string;
-  symbol: string;
-  decimals?: number;
-  tokenAddress: string;
-  faucetInitialLiquidity?: boolean;
-};
+export type TokenConfig =
+  | {
+      isMock: true;
+      name?: string;
+      symbol: string;
+      decimals?: number;
+      faucetInitialLiquidity?: boolean;
```

Comment:
We can remove or set a fixed boolean value in `faucetInitialLiquidity` as for mainnet it will always be false.

Reply:
removed